ui/components/app/edit-gas-display/index.scss

```diff
@@ -21,10 +12,6 @@
     }
   }
 
-  &__dapp-acknowledgement-warning {
-    margin-bottom: 20px;
-  }
```

Comment:
No longer using this classname
![Screenshot 2023-07-28 at 4 08 41 PM](https://github.com/MetaMask/metamask-extension/assets/8112138/bfd2f375-3bb2-4d91-858d-7ac93ffe7902)